### PR TITLE
Merge dev into live

### PR DESCRIPTION
**Pull request by dalepotter**
Merges all changes in the current sprint on tools into the production (i.e. `master`) branch:

- Ports front-end to [Bootstrap](http://getbootstrap.com/)
- Implements the [Chosen jQuery plugin](https://harvesthq.github.io/chosen/) for nicer select fields
- Generates a Datastore link using javascript, where browser support exists
- Improves UI and adds clearer copy
- Adds additional fields to the query builder, including start and end dates
- Various other small fixes

Once merged, this will be deployed to http://datastore.iatistandard.org/query The existing `live` branch will then be deleted, leaving the `master` branch as the new production stream.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -2,254 +2,709 @@
...
+    var reportingOrgTypes = QueryBuilder.LinkGen.getMultiSelectInfo('18398991', 'reporting-org.type');
+    var sectors = QueryBuilder.LinkGen.getMultiSelectInfo('1954968791', 'sector');
+    var recipientCountries = QueryBuilder.LinkGen.getMultiSelectInfo('605980212', 'recipient-country');
+    var recipientRegions = QueryBuilder.LinkGen.getMultiSelectInfo('1179181326', 'recipient-region');
+    var start_date__lt = QueryBuilder.LinkGen.getCalendarDate('start_date__lt', 'start-date__lt');
+    var start_date__gt = QueryBuilder.LinkGen.getCalendarDate('start_date__gt', 'start-date__gt');
+    var end_date__lt = QueryBuilder.LinkGen.getCalendarDate('end_date__lt', 'end-date__lt');
+    var end_date__gt = QueryBuilder.LinkGen.getCalendarDate('end_date__gt', 'end-date__gt');
+    var transactionProviderOrg = QueryBuilder.LinkGen.getMultiSelectInfo('transaction_provider_org', 'transaction_provider-org');
+    var participating_org = QueryBuilder.LinkGen.getMultiSelectInfo('participating_org', 'participating_org');
```
Comment (allthatilk) [inline templates/index.html]: should be `participating-org`

File: templates/index.html [hunk 2]
```diff
@@ -2,254 +2,709 @@
...
+      reportingOrgTypes,
+      sectors,
+      recipientCountries,
+      recipientRegions,
+      start_date__lt,
+      start_date__gt,
+      end_date__lt,
+      end_date__gt,
+      transactionProviderOrg,
+      participating_org,
```
Comment (allthatilk) [inline templates/index.html]: As above

File: templates/index.html (hunk 1 above)
Comment (andylolz) [reply templates/index.html]: why?

Comment (allthatilk) [reply templates/index.html]: [see here](http://datastore.iatistandard.org/docs/api/#participating-org)

Comment (andylolz) [reply templates/index.html]: Good spot!

**Pull request merged**